Week-1/index.html

```diff
@@ -0,0 +1,83 @@
+    <title>Cattyboard</title>
+</head>
+<body>
+    <div class="main-container">
+        <div class="sidenav">
+            <ul>
+                <li><i class="las la-bars"></i></li>
+                <li><i class="las la-home"></i></li>
+                <li><i class="las la-search"></i></li>
+                <li><i class="las la-volume-up"></i></li>
+                <li><i class="las la-user"></i></li>
+                <li><i class="las la-cog"></i></li>
+                <li><i class="lab la-spotify"></i></li>
+                <li><i class="lab la-soundcloud"></i></li>
+              </ul>
+        </div>
+        <div class="container">
+            <div class="box"><img id="catty-image" src="https://vif1g.csb.app/src/assets/cattyboard.jpg" alt="Catty Album">
+            </div>
+            <div class="content">
+                <p id="description">Cattyboard top 100 single charts(11.06.36)</p>
+                <p id="artist">Unknown Artist</p>
+                <p id="details">2016.Charts.100 Songs</p>
+                <br><br><br><br><br>
+                <div id="head-controls">
+                    <span class="icon-list"><i class="las la-play"></i> Play all</span>
+                    <span class="icon-list"><i class="las la-plus"></i>Add to</span>
+                    <span class="icon-list"><i class="las la-ellipsis-h"></i>More</span>
+                </div>
+            </div>
+        </div>
+        <div class="tracks">
+            <table>
+              <tr>
+                <td>One Dance</td>
+                <td>Crake feat Crakekid & Cyla</td>
+                <td>2:54</td>
+                <td><a class="table-link" href="#">CATTY CLOUD SYNC</a></td>
+              </tr>
+              <tr>
+                <td>Panda</td>
+                <td>Catte</td>
+                <td>4:08</td>
+                <td><a class="table-link" href="#">CATTY CLOUD SYNC</a></td>
+              </tr>
+              <tr>
+                <td>Can't Stop the Feeling</td>
+                <td>Justin Timberlake</td>
+                <td>3:56</td>
+                <td><a class="table-link" href="#">CATTY CLOUD SYNC</a></td>
+              </tr>
+              <tr>
+                <td>Work from Home</td>
+                <td>Fifth Harmony ft. Deepak</td>
+                <td>3:34</td>
+                <td><a class="table-link" href="#">CATTY CLOUD SYNC</a></td>
+              </tr>
+            </table>
+          </div>    
+        <div class="footer">
+            <div>
+                <img  class="footer-img" src="https://vif1g.csb.app/src/assets/cattyboard.jpg" alt="catty">
+            </div>
+            <div class="song">
+                <p>If It Ain't Love</p>
+                <p>Catson Derula</p>
+            </div>
+            <div class="foot-controls">
+                <span class="footer-list"><i class="las la-step-backward"></i></span>
+                <span class="footer-list"><i class="las la-step-forward"></i></span>
```

Comment:
lets follow BEM naming convention for classes plz refer http://getbem.com/naming/